### PR TITLE
[CIS-610] Message list UI polishing

### PR DESCRIPTION
**Pull request by mantergo**
**In this PR:** 

- Adjust navigation bar layout on `ChatVC`

Before / After
Screenshot 2021-01-28 at 13 43 43  Screenshot 2021-01-28 at 14 32 13

- Change the layout of the `ChatMessageComposerQuoteBubbleView`

Before / After

Screenshot 2021-01-28 at 13 45 43     Screenshot 2021-01-28 at 13 36 21

Before / After

Screenshot 2021-01-28 at 13 44 10 Screenshot 2021-01-28 at 13 36 05

### CODE REVIEW

File: Sources/StreamChatUI/ChatMessageList/MessageComposer/ChatMessageComposerQuoteBubbleView.swift [hunk 1]
```diff
@@ -61,53 +69,79 @@ open class _ChatMessageComposerQuoteBubbleView<ExtraData: ExtraDataTypes>: View,
...
-        attachmentPreview.widthAnchor.pin(equalToConstant: attachmentPreviewWidth).isActive = true
-        attachmentPreview.heightAnchor.pin(equalTo: attachmentPreview.widthAnchor, multiplier: 1).isActive = true
-
-        container.centerStackView.addArrangedSubview(textView)
+        containerConstraints = [
+            container.leadingAnchor.pin(equalTo: leadingAnchor, constant: directionalLayoutMargins.leading),
+            container.trailingAnchor.pin(equalTo: trailingAnchor, constant: -directionalLayoutMargins.trailing),
+            container.topAnchor.pin(equalTo: topAnchor, constant: directionalLayoutMargins.top),
+            container.bottomAnchor.pin(equalTo: bottomAnchor, constant: -directionalLayoutMargins.bottom)
+        ]
```
Comment (mantergo) [inline Sources/StreamChatUI/ChatMessageList/MessageComposer/ChatMessageComposerQuoteBubbleView.swift:94]: We can pin the constraints to `layoutMarginsGuide` but in this case it will be harder to customize it with just nullifying or changing constants.

File: Sources/StreamChatUI/ChatMessageList/ChatVC.swift [hunk 2]
```diff
@@ -146,14 +146,14 @@ open class _ChatVC<ExtraData: ExtraDataTypes>: ViewController,
...
-        subtitle.font = uiConfig.font.subheadline
+        subtitle.font = uiConfig.font.footnote
         subtitle.textColor = uiConfig.colorPalette.subtitleText
 
         let titleView = UIStackView(arrangedSubviews: [title, subtitle])
         titleView.axis = .vertical
-        titleView.spacing = 2
 
         navigationItem.titleView = titleView
+        navigationController?.navigationBar.topItem?.title = ""
```
Comment (VojtaStavik) [inline Sources/StreamChatUI/ChatMessageList/ChatVC.swift:156]: Why is this needed? This doesn't look exactly safe to modify someone else's navigation item :)

**Review by VojtaStavik — COMMENTED** [review]
Great work with the quoted message 👍 I wouldn't worry about the implementation details now because this part will be most likely heavily refactored soon anyway.

Do you think the navbar changes are enough? Shouldn't the font be even smaller? 🤔

File: Sources/StreamChatUI/ChatChannelList/ChatChannelListVC.swift [hunk 3]
```diff
@@ -58,6 +58,8 @@ open class _ChatChannelListVC<ExtraData: ExtraDataTypes>: ViewController,
         
         controller.setDelegate(self)
         controller.synchronize()
+        
+        navigationItem.backButtonTitle = ""
```
Comment (VojtaStavik) [inline Sources/StreamChatUI/ChatChannelList/ChatChannelListVC.swift:62]: I just realized `ChatChannelListVC` doesn't use the lifecycle methods. We'll fix that later...

**Review by VojtaStavik — APPROVED** [review]
🚢 🇮🇹
